hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/AbstractStateMachineTableProcedure.java

```diff
@@ -192,4 +192,15 @@ protected static void checkOnline(MasterProcedureEnv env, RegionInfo ri)
     }
     regionNode.checkOnline();
   }
+
+  /**
+   * Some procedures cannot safely run while a table is being modified. Helper method to allow them
+   * to check if such a procedure is running on a table.
+   */
+  protected boolean isTableModificationInProgress(MasterProcedureEnv env) {
```

Comment:
Make it final which means it is method to be called by sub classes, not be overidden?

Reply:
done